lib/Jsrt/ChakraCommon.h

```diff
@@ -2978,4 +2978,343 @@ typedef unsigned char* ChakraBytePtr;
         JsSetPromiseContinuationCallback(
             _In_ JsPromiseContinuationCallback promiseContinuationCallback,
             _In_opt_ void *callbackState);
+
+    /////////////////////
+
+    /// <summary>
+    ///     TTD API -- may change in future versions:
```

Comment:
We should use a <remarks>section just after <summary> to note that these APIs are experimental and may change in future. Or we put these in a new header file and only enable under a macro say USE_CHAKRA_TTD_EXPIREMENTAL_APIS